diff --git a/cloud-init/cloudinit/CloudConfig/cc_set_passwords.py b/cloud-init/cloudinit/CloudConfig/cc_set_passwords.py
index 9d0bbdb..e98c881 100644
--- a/cloud-init/cloudinit/CloudConfig/cc_set_passwords.py
+++ b/cloud-init/cloudinit/CloudConfig/cc_set_passwords.py
@@ -99,7 +99,7 @@ def handle(_name, cfg, _cloud, log, args):
         msg = "set PasswordAuthentication to '%s'" % pw_auth
         try:
             cmd = ['sed', '-i', 's,%s,\\2%s,' % (pa_s, pw_auth),
-                   '/etc/ssh/sshd_config']
+                   '/etc/openssh/sshd_config']
             util.subp(cmd)
             log.debug(msg)
         except Exception as e:
